unit/Makefile

```diff
@@ -60,7 +60,7 @@ SRC += analyses/ai/ai.cpp \
        util/simplify_expr.cpp \
        util/small_map.cpp \
        util/small_shared_two_way_ptr.cpp \
-			 util/std_expr.cpp \
+       util/std_expr.cpp \
```

Comment:
I'd recommend putting such commits in a separate PR, this would likely have been merged weeks ago...

Reply:
Noted. Will split this into separate PR now.